src/pipeline_index.py

```diff
@@ -0,0 +1,65 @@
+"""Fondant pipeline to index a RAG system."""
+import pyarrow as pa
+from fondant.pipeline import Pipeline
+
+
+def create_pipeline(
```

Comment:
```suggestion
def create_pipeline(  # noqa: PLR0913
```

To suppress the ruff error.